evennia/server/inputfuncs.py

```diff
@@ -421,3 +421,62 @@ def unmonitor(session, *args, **kwargs):
     """
     kwargs["stop"] = True
     monitor(session, *args, **kwargs)
+
+
+def _on_webclient_options_change(**kwargs):
+    """
+    Called when the webclient options stored on the player changes.
+    Inform the interested clients of this change.
+    """
+    session = kwargs["session"]
```

Comment:
This probably needs to be checked for existence before continuing (`if session: ...`); there are a few cases (such as abrupt client disconnect at *just* the right moment) where session may be `None`.